flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/FileSystemITCaseBase.scala

```diff
@@ -95,6 +95,60 @@ trait FileSystemITCaseBase {
          |)
        """.stripMargin
     )
+
+    tableEnv.executeSql(
+      s"""
+         |create table hasDecimalFieldWithPrecisionTenAndZeroTable (
+         |  x decimal(10, 0), y int
+         |) with (
+         |  'connector' = 'filesystem',
+         |  'path' = '$resultPath',
+         |  ${formatProperties().mkString(",\n")}
+         |)
+       """.stripMargin
+    )
+
+    tableEnv.executeSql(
+      s"""
+         |create table hasDecimalFieldWithPrecisionThreeAndTwoTable (
+         |  x decimal(3, 2), y int
+         |) with (
+         |  'connector' = 'filesystem',
+         |  'path' = '$resultPath',
+         |  ${formatProperties().mkString(",\n")}
+         |)
+       """.stripMargin
+    )
+  }
+
+  @Test
+  def testSelectDecimalWithPrecisionTenAndZeroFromFileSystem(): Unit={
```

Comment:
"FromCsv" is more correct

Reply:
This is because now all test case is in Base, the specific subClass define the implements from avo, csv, json and etc.So if I follow the test behavior before, I think the name "FromFileSystem" is suit for this function. However, if I take this case to  "FileSystemTestCsvITCase" or "StreamFileSystemTestCsvITCase", i need to rewrite the same test case twice time(and now subClass that contains these two class doesn't have any test cases at all).

Reply:
make sense